### PR TITLE
Externalized commands titles and added categories.

### PR DESCRIPTION
**Pull request by HCRenegade**
Preview
qciz0vopcr

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,565 +1,577 @@
-{
```
Comment (ruslan-bikkinin) [inline package.json]: Could you please fix indentation issues in this file?

File: package.json [hunk 2]
```diff
@@ -1,565 +1,577 @@
...
+			},
+			{
+				"command": "reactNative.reloadApp",
+				"title": "%reactNative.command.reloadApp.title%",
+				"category": "React Native"
+			},
+			{
+				"command": "reactNative.runInspector",
+				"title": "%reactNative.command.runInspector.title%",
+				"category": "React Native:"
```
Comment (ruslan-bikkinin) [inline package.json]: Could you please remove `:` in category section for consistency with other entries?

**Comment by HCRenegade** [comment]
Yup sure, will do that in a while. :)

**Comment by HCRenegade** [comment]
@SounD120 @ruslan-bikkinin should have fixed everything, hope everything is okay 😄

**Comment by HCRenegade** [comment]
Alright I'm having some troubles with git. Everything should be ok in a while.

**Pull request closed without merging**